src/Microsoft.PowerShell.Commands.Utility/commands/utility/NewTimeSpanCommand.cs

```diff
@@ -68,25 +68,25 @@ public DateTime End
         /// Allows the user to override the day.
         /// </summary>
         [Parameter(ParameterSetName = "Time")]
-        public int Days { get; set; } = 0;
+        public int Days { get; set; }
```

Comment:
@iSazonov Revert such changes or not?

Reply:
No, null and 0 is obvious, we should remove them all.

Reply:
If we have many initializations with enums perhaps we could set "suggestion" for the rule to exclude many suppressions.